client/extensions/woocommerce/index.js

```diff
@@ -221,10 +221,23 @@ export default function() {
 	} );
 
 	// Add pages that use my-sites navigation instead
-	page( '/store/stats/:type/:unit', controller.siteSelection, controller.sites );
-	page( '/store/stats/:type/:unit/:site', siteSelection, navigation, StatsController );
+	page(
+		'/store/stats/:type/:unit',
+		controller.siteSelection,
+		controller.sites,
```

Comment:
Both of these are recurring controllers in `my-sites` -- we need to make sure they're STR compliant, or provide an STR-compliant counterpart.